lib/griddler/mailgun/adapter.rb

```diff
@@ -80,10 +80,10 @@ def param_or_header(key)
       end
 
       def attachment_files
-        attachment_count = params['attachment-count'].to_i
+        attachment_count = params['attachment_count'].to_i
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.